### ISSUE TITLE
User can not have more than two disease limitation 

### ISSUE THREAD

**Issue by foyiri** [hzi-braunschweig/SORMAS-Project#1308]
I just notice that you can not add multiple disease to a particular user i was trying to add measles and yellow fever to a user account

**Comment by Iheanacho2027** [comment]
Yes this is very important, there should be limitations to more than one priority disease, not just one disease as it is on the system.
